data/exercises/031_is_prime.md

```diff
@@ -2,23 +2,23 @@
 title: Determine Whether a Given Integer Number Is Prime
 slug: "31"
 difficulty: intermediate
-tags: [ "arithmetic" ]
+tags: ["arithmetic"]
 description: "Check if a given integer is a prime number."
 ---
 
 # Solution
 
-  Recall that `d` divides `n` iff `n mod d = 0`.  This is a naive
-  solution.  See the [Sieve of
-  Eratosthenes](http://en.wikipedia.org/wiki/Sieve_of_Eratosthenes) for a
-  more clever one.
-  
+Recall that `d` divides `n` iff `n mod d = 0`. This is a naive
```

Comment:
Should "iff" be "if" or does it stand for "if and only if"? If the latter, it should be IFF in all caps to as no to cause confusion.

Reply:
> Should "iff" be "if" or does it stand for "if and only if"? If the latter, it should be IFF in all caps to as no to cause confusion.

I think it be should IFF since it is explaining that the condition for d to divide n is that the remainder of n divided by d is zero. Let me change it in caps then.